README.md

```diff
@@ -41,6 +41,12 @@ Wikis can not been named *within* Patch demo. Wikis are listed within Patch demo
 
 These will be considered on a case-by-case basis, but will generally be allowed as long as they don't interfere with other teams' ability to test in a production-like environment.
 
+**What if I don't like the above?**
```

Comment:
Or "What if I need more control?"

Reply:
I mean, we don't need to be that serious…

Reply:
"the above restrictions"?